deepchem/models/torch_models/layers.py

```diff
@@ -4187,3 +4189,200 @@ def gaussian_histogram(self, x: torch.Tensor) -> torch.Tensor:
         output = output / torch.sum(output, dim=2, keepdim=True)
         output = output.view(-1, self.n_input * 11)
         return output
+
+
+class ResidualBlock(nn.Module):
```

Comment:
@rbharath Do you want me to add ResidualBlock as a separate layer or a helper class just for MXMNet layers?

Reply:
A separate layer I think. Please go ahead and add the usage example

Reply:
I found out that by making a little change in MultilayerPerceptron, it can be used as ResidualBlock as well. I have made a PR for that, can you please review it? 
@rbharath @vsomnath @ARY2260

Reply:
Link - https://github.com/deepchem/deepchem/pull/3494